docs/source/api/deliveryservices_id_routing.rst

```diff
@@ -30,7 +30,7 @@ Retrieves the aggregated routing percentages for a given :term:`Delivery Service
 .. versionchanged:: 1.5
 	Older versions of this endpoint had different behaviors in handling non HTTP/DNS :term:`Delivery Services` and :term:`Delivery Service` regexps.
 	It used to return an empty response object if the :term:`Delivery Service` being requested did not have a HTTP or DNS type. It will now return an error.
-	For :term:`Delivery Service` regexps it used to use all regexps to compare host names in the routing stats which was incorrect. It will now only compare :term:`Delivery Service` host regexps.
+	For :term:`Delivery Service` regexps it used to use all regexps to compare host names in the routing stats which was incorrect. It will now only compare :term:`Delivery Service` host regular expression(s).
```

Comment:
you've got "regexps" in here three more times

Reply:
ha